packages/docusaurus-plugin-pwa/package.json

```diff
@@ -39,11 +39,11 @@
     "workbox-window": "^6.1.1"
   },
   "devDependencies": {
-    "@docusaurus/module-type-aliases": "2.0.0-beta.9",
+    "@docusaurus/module-type-aliases": "workspace:*",
     "fs-extra": "^10.0.0"
   },
   "peerDependencies": {
-    "@babel/core": "^7.0.0",
+    "@babel/core": ">=7.0.0",
```

Comment:
what is a reason of target change?

this range is not correct, we should not allow babel@8.0.0 when it gets released

Reply:
You're right, this is a mistake.